ADALiOS/ADALiOSTests/ADInstanceDiscoveryTests.m

```diff
@@ -431,11 +433,10 @@ -(void) testNonValidatedAuthority
 {
     [self adSetLogTolerance:ADAL_LOG_LEVEL_ERROR];
     NSUUID* correlationId = [NSUUID UUID];
-    [self validateAuthority:@"https://MyFakeAuthority.com/MSOpenTechBV.onmicrosoft.com" correlationId:correlationId line:__LINE__];
+    [self validateAuthority:@"https://MyFakeAuthority.microsoft.com/MSOpenTechBV.onmicrosoft.com" correlationId:correlationId line:__LINE__];
     XCTAssertFalse(mValidated);
```

Comment:
I look forward to the day that someone registers myfakeauthority.microsoft.com for something.